users/src/test/java/io/crate/operation/auth/AuthenticationServiceTest.java

```diff
@@ -181,4 +212,34 @@ public void testMatchAddress() throws Exception {
         assertTrue(isValidAddress(entry,
             String.format("%s.%s.%s.%s", randomInt(255), randomInt(255), randomInt(255), randomInt(255))));
     }
+
+    @Test
+    public void testConvertHBASetting() throws Exception {
+        Settings settings = Settings.builder()
+            .put("auth.host_based.0.user", "crate")
+            .put("auth.host_based.0.address", "127.0.0.1")
+            .put("auth.host_based.0.method", "trust")
+            .put("auth.host_based.1.user", "crate")
+            .put("auth.host_based.1.address", "0.0.0.0/0")
+            .put("auth.host_based.1.method", "fake")
+            .build();
+
+
+        SQLPlugin sqlPlugin = new SQLPlugin(Settings.EMPTY);
```

Comment:
pack it into a `createClusterService(Settings settings)` method and re-use it also for the setup method